### DISCUSSION TITLE
Socket error delegate

### DISCUSSION THREAD

**Discussion by gregoireLem** [HaishinKit/HaishinKit.swift#1431] [Q&A]
Is there a way to get notified from the client about what causing the connexion error ?
Like "wrong passphrase" for example ? 
I can see the print but no delegate available to bind the error

**Comment by shogo4405** [comment]
That's not available in the SRTHaishinKit layer. I found the following code in the original SRT sample code, so it seems like it could be ported: https://github.com/Haivision/srt/blob/f99ce57760aefb2cd21af275e8de0fff9132c6e3/examples/sendmsg.cpp#L67-L72
